src/integrations/prefect-ray/tests/test_task_runners.py

```diff
@@ -371,6 +372,24 @@ def test_flow():
         assert bx.type == StateType.PENDING
         assert cx.type == StateType.COMPLETED
 
+    def test_as_completed_yields_correct_order(self, task_runner):
+        @task
+        def task_a(seconds):
+            time.sleep(seconds)
+            return seconds
+
+        timings = [1, 5, 10]
+
+        @flow(version="test", task_runner=task_runner)
+        def test_flow():
+            done_futures = []
+            futures = [task_a.submit(seconds) for seconds in timings]
```

Comment:
same comment as above
```suggestion
            futures = [task_a.submit(seconds) for seconds in reversed(timings)]
```